### PR TITLE
fix(react-utilities): add microdata properties like itemType to allowed html properties

### PR DESCRIPTION
**Pull request by YuanboXue-Amber**
Add the below properties to allowed properties for every html element:
[itemid](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/itemid)
[itemprop](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/itemprop)
[itemref](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/itemref)
[itemscope](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/itemscope)
[itemtype](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/itemtype)

### CODE REVIEW

File: packages/react-components/react-utilities/src/utils/properties.ts
Comment (bsunderhus) [inline packages/react-components/react-utilities/src/utils/properties.ts:134]: nit: would be nice to add a link to MDN about microdata and its attributes
https://developer.mozilla.org/en-US/docs/Web/HTML/Microdata

**Review by bsunderhus — APPROVED** [review]
LGTM 👍🏻

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                 Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `79.349 kB`<br />`24.053 kB` |  `79.407 kB`<br />`24.083 kB` | `58 B` <br />`30 B`  |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `83.788 kB`<br />`20.843 kB` |  `83.846 kB`<br />`20.873 kB` | `58 B` <br />`30 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `48.281 kB`<br />`13.647 kB` |  `48.339 kB`<br />`13.676 kB` | `58 B` <br />`29 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |    `14.85 kB`<br />`5.942 kB` |   `14.908 kB`<br />`5.971 kB` | `58 B` <br />`29 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |   `68.249 kB`<br />`18.99 kB` |  `68.307 kB`<br />`19.019 kB` | `58 B` <br />`29 B`  |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |     `22.5 kB`<br />`7.158 kB` |   `22.558 kB`<br />`7.187 kB` | `58 B` <br />`29 B`  |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |    `23.403 kB`<br />`7.45 kB` |   `23.461 kB`<br />`7.478 kB` | `58 B` <br />`28 B`  |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `23.947 kB`<br />`7.022 kB` |   `24.006 kB`<br />`7.049 kB` | `59 B` <br />`27 B`  |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `36.396 kB`<br />`9.579 kB` |   `36.454 kB`<br />`9.608 kB` | `58 B` <br />`29 B`  |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `43.469 kB`<br />`10.812 kB` |   `43.527 kB`<br />`10.84 kB` | `58 B` <br />`28 B`  |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `39.014 kB`<br />`10.456 kB` |  `39.072 kB`<br />`10.485 kB` | `58 B` <br />`29 B`  |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |   `46.544 kB`<br />`11.84 kB` |   `46.602 kB`<br />`11.87 kB` | `58 B` <br />`30 B`  |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |   `51.91 kB`<br />`11.003 kB` |  `51.968 kB`<br />`11.031 kB` | `58 B` <br />`28 B`  |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `67.458 kB`<br />`19.264 kB` |  `67.516 kB`<br />`19.291 kB` | `58 B` <br />`27 B`  |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |   `63.14 kB`<br />`18.176 kB` |  `63.198 kB`<br />`18.202 kB` | `58 B` <br />`26 B`  |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `8.461 kB`<br />`3.555 kB` |    `8.519 kB`<br />`3.582 kB` | `58 B` <br />`27 B`  |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |    `9.504 kB`<br />`3.896 kB` |    `9.562 kB`<br />`3.922 kB` | `58 B` <br />`26 B`  |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |     `8.562 kB`<br />`3.61 kB` |     `8.62 kB`<br />`3.639 kB` | `58 B` <br />`29 B`  |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `73.757 kB`<br />`23.969 kB` |  `73.815 kB`<br />`24.003 kB` | `58 B` <br />`34 B`  |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |  `73.344 kB`<br />`23.876 kB` |  `73.402 kB`<br />`23.909 kB` | `58 B` <br />`33 B`  |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `188.816 kB`<br />`51.903 kB` | `188.874 kB`<br />`51.932 kB` | `58 B` <br />`29 B`  |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `33.259 kB`<br />`10.952 kB` |  `33.317 kB`<br />`10.983 kB` | `58 B` <br />`31 B`  |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |  `85.361 kB`<br />`25.458 kB` |   `85.42 kB`<br />`25.487 kB` | `59 B` <br />`29 B`  |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |   `16.359 kB`<br />`5.853 kB` |   `16.417 kB`<br />`5.883 kB` | `58 B` <br />`30 B`  |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |    `10.68 kB`<br />`4.215 kB` |   `10.738 kB`<br />`4.245 kB` | `58 B` <br />`30 B`  |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `23.498 kB`<br />`7.617 kB` |   `23.556 kB`<br />`7.642 kB` | `58 B` <br />`25 B`  |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |    `9.238 kB`<br />`3.815 kB` |    `9.296 kB`<br />`3.843 kB` | `58 B` <br />`28 B`  |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `12.231 kB`<br />`4.925 kB` |   `12.289 kB`<br />`4.953 kB` | `58 B` <br />`28 B`  |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `115.697 kB`<br />`35.316 kB` | `115.756 kB`<br />`35.348 kB` | `59 B` <br />`32 B`  |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `118.896 kB`<br />`35.806 kB` | `118.955 kB`<br />`35.833 kB` | `59 B` <br />`27 B`  |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `102.835 kB`<br />`31.498 kB` | `102.894 kB`<br />`31.525 kB` | `59 B` <br />`27 B`  |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `15.655 kB`<br />`5.835 kB` |   `15.713 kB`<br />`5.864 kB` | `58 B` <br />`29 B`  |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |  `36.025 kB`<br />`11.914 kB` |  `36.083 kB`<br />`11.944 kB` | `58 B` <br />`30 B`  |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |   `14.148 kB`<br />`5.654 kB` |   `14.206 kB`<br />`5.683 kB` | `58 B` <br />`29 B`  |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |   `20.746 kB`<br />`7.299 kB` |   `20.804 kB`<br />`7.327 kB` | `58 B` <br />`28 B`  |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |   `32.07 kB`<br />`10.033 kB` |   `32.128 kB`<br />`10.06 kB` | `58 B` <br />`27 B`  |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `43.843 kB`<br />`12.336 kB` |  `43.901 kB`<br />`12.364 kB` | `58 B` <br />`28 B`  |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |   `19.877 kB`<br />`6.392 kB` |    `19.935 kB`<br />`6.42 kB` | `58 B` <br />`28 B`  |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |  `32.562 kB`<br />`10.253 kB` |   `32.62 kB`<br />`10.282 kB` | `58 B` <br />`29 B`  |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `11.682 kB`<br />`4.561 kB` |    `11.74 kB`<br />`4.587 kB` | `58 B` <br />`26 B`  |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |   `14.992 kB`<br />`4.995 kB` |    `15.05 kB`<br />`5.025 kB` | `58 B` <br />`30 B`  |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |    `23.674 kB`<br />`7.83 kB` |    `23.732 kB`<br />`7.86 kB` | `58 B` <br />`30 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                          |         Size (minified/GZIP) |
| -------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                 |         `533 B`<br />`341 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr> |         `554 B`<br />`348 B` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                            |  `10.685 kB`<br />`4.104 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                 |  `10.576 kB`<br />`3.875 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                 |    `19.7 kB`<br />`7.404 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                              | `41.502 kB`<br />`14.623 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                    |         `180 B`<br />`159 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a85b5f8ff499fddcb1edcee28c7b0dede92ab674'>a85b5f8ff499fddcb1edcee28c7b0dede92ab674</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/Avatar-mount_base.html">1311</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/Avatar-mount.html">1248</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/Button-mount_base.html">848</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/Button-mount.html">879</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/FluentProvider-mount_base.html">1410</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/FluentProvider-mount.html">1539</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/FluentProviderWithTheme-mount_base.html">622</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/FluentProviderWithTheme-mount.html">628</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">516</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">527</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">625</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">633</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/MakeStyles-mount_base.html">1882</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/MakeStyles-mount.html">1904</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/SpinButton-mount_base.html">2566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24652/perf-test-react-components/SpinButton-mount.html">2328</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by layershifter — APPROVED** [review]

**Review by miroslavstastny — APPROVED** [review]

File: packages/react-components/react-utilities/src/utils/properties.ts
Comment (YuanboXue-Amber) [reply packages/react-components/react-utilities/src/utils/properties.ts:134]: Good idea. Too bad I didn't see this before the hitting the merge button hard 🙈

**Pull request merged**
